src/melee.cpp

```diff
@@ -486,6 +486,13 @@ void player::melee_attack( Creature &t, bool allow_special, const matec_id &forc
             technique_id = tec_none;
         }
 
+        // if you have two broken arms you aren't doing any martial arts
+        // and your hits are not going to hurt very much
+        if( get_working_arm_count() < 1 ) {
+            technique_id = tec_none;
+            d.mult_damage( .1 );
```

Comment:
```suggestion
            d.mult_damage( 0.1 );
```
to me this makes it more obvious that it's a decimal.